consensus/src/pipeline/buffer_manager.rs

```diff
@@ -272,7 +272,7 @@ impl BufferManager {
             .find_elem_from(cursor.or_else(|| *self.buffer.head_cursor()), |item| {
                 item.is_ordered()
             });
-        info!(
+        debug!(
```

Comment:
I'm not sure if we want to change any of these logs in this file to debug!. They are extremely useful today for debugging.

Reply:
Sounds good. Will wait for others to confirm, and if so, we can rollback this file 😄

Reply:
+1, but we can try to minimize the output, for the commit_info when there's epoch state inside

Reply:
SGTM 😄 Just did the simple thing for now.